### DISCUSSION TITLE
Support for React 19

### DISCUSSION THREAD

**Discussion by DenisBessa** [mantinedev/mantine#6316] [Q&A]
First of all, I can't express how much I like Mantine. I think it's just perfect, and is the best UI Library one can imagine.

That said, I was wondering what are the plans for supporting React 19. I think it probably will have an awesome impact on performance.

**Comment by BrendanC23** [comment]
Bump.

**Comment by IsaacInsoll** [comment]
I'd love to know the roadmap for React 19 support in Mantine too

**Comment by rtivital** [comment]
React 19 has not been released yet. The support will be provided after the official React 19 release.

**Reply by mamlzy** [reply]
> React 19 has not been released yet. The support will be provided after the official React 19 release.

so @rtivital i can't use mantine with next 15? which is stable now..☹️

**Reply by rtivital** [reply]
See https://github.com/mantinedev/mantine/issues/7008

**Reply by comertcimen** [reply]
@rtivital React 19 stable is just released.

**Reply by rtivital** [reply]
So? Do you experience any issues with the latest version?

**Reply by comertcimen** [reply]
No, didn't know that it is already supported

**Comment by tonai** [comment]
React 19 has been released yesterday :)
Is there anything to do to migrate an existing code base using Mantine with React 18 ?
Thanks

**Reply by DenisBessa** [reply]
> React 19 has been released yesterday :) Is there anything to do to migrate an existing code base using Mantine with React 18 ? Thanks

Mantine is already compatible with React 19. You shouldn't find any trouble updating.
